builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -102,6 +102,15 @@ steps:
     targetFolder: $(azureFunctionsExtensionBundlePath)/bin
     overWrite: true
 
+  # Copy the Sql nupkg to ensure it's available for tests.
+- task: CopyFiles@2
```

Comment:
But why does the build not do the copy? We have the task there so duplicating it here seems weird.

Reply:
I've looked up why occasionally the package copy task isn't working as intended, but couldn't find anything relatable. I'll keep investigating meanwhile since this has happened a couple times, wanted to have this redundancy.

Reply:
Sounds good, could you add that to the comment so future us remembers why we have this duplicate copy?